frontend/server/src/Controllers/Contest.php

```diff
@@ -1177,6 +1177,7 @@ public static function getContestListDetailsv2ForTypeScript(
         return [
             'smartyProperties' => [
                 'payload' => [
+                    'query' => $r['query'],
```

Comment:
```suggestion
                    'query' => $r->ensureOptionalString('query'),
```

en ningún código nuevo se debería usar `$r[...]`.